quickstart/configs.go

```diff
@@ -22,24 +24,33 @@ func configBaseForm(quickstart *Quickstart) (*State, error) {
 		configFields := []huh.Field{
 			huh.NewInput().
 				Title("Choose an sdkClassName for your target:").
+				Placeholder("Your SDK object: _if you choose \"sdk\", your users will access methods like \"sdk.doThing()").
```

Comment:
I think my comment wasn't very clear. I really meant that I think the title should be `Name your SDK object` and the placeholder should be  `for example, if you choose "sdk", your users will write "sdk.doThing()"`